doc/Overview/Mvux/Overview.md

```diff
@@ -4,7 +4,15 @@ uid: Overview.Mvux.Overview
 
 # MVUX Overview
 
-**M**odel, **V**iew, **U**pdate, e**X**tended (**MVUX**) is a variation of the MVU design pattern that encourages unidirectional flow of immutable data, whilst adding support for the data binding capabilities that makes the MVVM pattern so powerful. In other words, MVUX brings together the best of the MVU and MVVM design patterns.
+**M**odel, **V**iew, **U**pdate, e**X**tended (**MVUX**) is a variation of the MVU design pattern that encourages unidirectional flow of immutable data, whilst adding support for the data binding capabilities. And that capabilitie makes the MVVM pattern so powerful. In other words, MVUX brings together the best of the MVU and MVVM design patterns.
+
+In summary MVUX utilizes its powerful code-generation engine to generate a bindable proxy for each Model, which holds the state information of the data, as well as a bindable proxy for entities where needed.
+The bindable proxies use as a bridge that enables immutable entities to work with the WinUI data-binding engine. 
+The states in the Model are monitored for data-binding changes, and in response to any change, **the objects are recreated fresh**, instead of their properties being changed.
+
+Thus, with less code typed, we can have a more efficient and practical result.
```

Comment:
> Thus, with less code typed, we can have a more efficient and practical result.

Let's expand on this line further. Is the result more _efficient_ or _practical_ solely from less developer effort? Maybe delineate between these two aspects.

Reply:
```suggestion
```